pkg/fission-cli/cmd/spec/spec.go

```diff
@@ -490,6 +490,10 @@ func (fr *FissionResources) Validate(c *cli.Context) error {
 		if strategy.ExecutorType == fv1.ExecutorTypeNewdeploy && strategy.SpecializationTimeout < fv1.DefaultSpecializationTimeOut {
 			log.Warn(fmt.Sprintf("SpecializationTimeout in function spec.InvokeStrategy.ExecutionStrategy should be a value equal to or greater than %v", fv1.DefaultSpecializationTimeOut))
 		}
+		fnTimeout := f.Spec.FunctionTimeout
+		if fnTimeout <= 0 {
```

Comment:
The flag description said ` If set to zero, by default it will wait of 60s for the response.`. So the condition here should be `fnTimeout < 0`.

Reply:
Done